test/integration/generated/bodyComplexWithTracing/src/operations/array.ts

```diff
@@ -44,25 +43,16 @@ export class ArrayImpl implements Array {
   async getValid(
     options?: ArrayGetValidOptionalParams
   ): Promise<ArrayGetValidResponse> {
-    const { span } = createSpan(
+    return tracingClient.withSpan(
       "BodyComplexWithTracing-getValid",
-      options || {}
+      options ?? {},
```

Comment:
I don't recall if that actually compiles or if typescript complains. What I've seen work well is on line 44 replacing `options?: ArrayGetValidOptionalParams` with `options: ArrayGetValidOptionalParams = {}` 

Do you think it's possible to generate the code that way?

Reply:
It is not complaining. Tried to change options to have a default value of `= {}` but the change is a bit more involved than I expected. The reason is that we share logic to generate the interface and overloads which don't support default values, so there would be a bit more work to get that going.

If you feel strongly about this we can definitely, invest to make the change. I think we could have it like this and file an issue to improve this by adding the default values.

Reply:
If the compiler is happy I'm happy 😄 

But yeah, I agree there's no need to invest energy for this change if what you have here compiles fine. Thanks!